spec/ids_please/basic_spec.rb

```diff
@@ -1,51 +1,49 @@
 require 'spec_helper'
 
 describe IdsPlease do
-
   recognazible_links = %w(
-      https://www.facebook.com/fb_acc
-      https://facebook.com/fb_acc2<U+200>
-      http://instagram.com/inst_acc
-      http://hi5.com/hi5_acc
-      http://www.hi5.com/profile.html?uid=12341234
-      http://pinterest.com/pinterest_acc
-      http://blogger-acc.blogspot.com
-      http://livejournal-acc.livejournal.com
-      http://livejournal-acc2.livejournal.ru
-      https://www.blogger.com/blogger.g?blogID=12341234#overview/src=dashboard
-      http://vk.com/vk_acc
-      http://linkedin.com/in/xnutsive
-      http://www.linkedin.com/company/evil-martians
-      http://www.linkedin.com/profile/view?id=12341234&trk=nav_responsive_tab_profile
-      http://Ameblo.jp/ameba_acc
-      http://reddit.com/user/reddit_acc
-      https://twitter.com/twi_acc
-      https://vimeo.com/vimeo_acc
-      https://plus.google.com/12341234
-      https://plus.google.com/+VladimirBokov
-      https://soundcloud.com/sc_acc
-      https://youtube.com/channels/yb_acc
-      http://tumblr-acc.tumblr.com
-      http://odnoklassniki.com/profile/12341234/about
-      http://ok.ru/profile/12341234/about
-      http://odnoklassniki.com/group/43214321/about?some=123
-      http://moikrug-acc.moikrug.ru
-      https://my.mail.ru/community/test-group-102/
-      https://my.mail.ru/mail/gazay/
-    )
+    https://www.facebook.com/fb_acc
+    https://facebook.com/fb_acc2<U+200>
+    http://instagram.com/inst_acc
+    http://hi5.com/hi5_acc
+    http://www.hi5.com/profile.html?uid=12341234
+    http://pinterest.com/pinterest_acc
+    http://blogger-acc.blogspot.com
+    http://livejournal-acc.livejournal.com
+    http://livejournal-acc2.livejournal.ru
+    https://www.blogger.com/blogger.g?blogID=12341234#overview/src=dashboard
+    http://vk.com/vk_acc
+    http://linkedin.com/in/xnutsive
+    http://www.linkedin.com/company/evil-martians
+    http://www.linkedin.com/profile/view?id=12341234&trk=nav_responsive_tab_profile
+    http://Ameblo.jp/ameba_acc
+    http://reddit.com/user/reddit_acc
+    https://twitter.com/twi_acc
+    https://vimeo.com/vimeo_acc
+    https://plus.google.com/12341234
+    https://plus.google.com/+VladimirBokov
+    https://soundcloud.com/sc_acc
+    https://youtube.com/channels/yb_acc
+    http://tumblr-acc.tumblr.com
+    http://odnoklassniki.com/profile/12341234/about
+    http://ok.ru/profile/12341234/about
+    http://odnoklassniki.com/group/43214321/about?some=123
+    http://moikrug-acc.moikrug.ru
+    https://my.mail.ru/community/test-group-102/
+    https://my.mail.ru/mail/gazay/
+  )
 
   not_recognazible_links = %w(
-      http://fucebook.com/not_recognized
-      http://vka.com/not_recognized
-    )
+    http://fucebook.com/not_recognized
+    http://vka.com/not_recognized
+  )
 
   not_parseble_links = %w(
-      http://vk.com
-      http://soundcloud.com
-    )
+    http://vk.com
+    http://soundcloud.com
+  )
 
   describe 'recognize' do
-
```

Comment:
Style preferences: if there one example in describe – I'm not using excess newline, but otherwise do